features/all/phpcr_query_update.feature

```diff
@@ -153,3 +153,30 @@ Feature: Execute a a raw UPDATE query in JCR_SQL2
             | UPDATE [nt:unstructured] mixin_foo('bar') |
             | UPDATE [nt:unstructured] APPLY mixin_foo('bar') |
             | UPDATE [nt:unstructured] mixin_foo'bar') |
+
+    Scenario Outline: Execute update query with expressions
+        Given I execute the "<query>" command
+        Then the command should not fail
+        And I save the session
```

Comment:
I think the main issue is that it is necessary to execute 2 commands (the second one is to save the session). The first command should not fail - if it did fail the second command (session:save) would still succeed.

So I need to indicate that the first command should or should not fail before saving the session.

I suppose it would make sense to combine the session save in a single statement:

````cucumber
When I run the "UPDATE ..." command and save the session
Then the command should not fail
And the node at "/foo" should exist
And I should see the following:
"""
foo
"""
````